### PR TITLE
Added SQL ELT Application to CDAP development build

### PR DESCRIPTION
**Pull request by dli357**
Added submodule to automatically build SQL pipelines into CDAP.

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -18,3 +18,8 @@
 	path = app-artifacts/dre
 	url = ../../cdap-solutions/dre.git
 	branch = develop
+[submodule "app-artifacts/sql-pipelines"]
+	path = app-artifacts/sql-pipelines
+	url = https://github.com/cdapio/sql-pipelines.git
```
Comment (dereklwood) [inline .gitmodules]: use relative url for consistency

File: .gitmodules [hunk 2]
```diff
@@ -18,3 +18,8 @@
...
+	branch = develop
+
```
Comment (dereklwood) [inline .gitmodules]: remove blank line

File: .gitmodules (hunk 1 above)
Comment (dli357) [reply .gitmodules]: Fixed

File: .gitmodules (hunk 2 above)
Comment (dli357) [reply .gitmodules]: Fixed

**Pull request merged**